combustion-android-ble/src/main/java/inc/combustion/framework/ble/WiFiNodesManager.kt

```diff
@@ -54,32 +46,23 @@ import kotlin.random.Random
 import kotlin.random.nextUInt
 
 internal class WiFiNodesManager(
+    private val scope: CoroutineScope,
     private val getNodeDevice: (deviceId: String) -> NodeBleDevice,
 ) {
     private val connectedWiFiNodes = StateFlowMutableMap<String, NodeBleDevice>()
     private val nonWiFiNodes: MutableSet<String> = mutableSetOf()
     private val nodeToMutexMap: MutableMap<String, Semaphore> = ConcurrentHashMap()
-    private val coroutineScope = CoroutineScope(SupervisorJob())
 
-    private lateinit var lifecycleChildScope: CoroutineScope
-    private lateinit var lifecycleChildJob: Job
-
-    fun initialize(owner: LifecycleOwner) {
-        lifecycleChildJob = SupervisorJob()
-        lifecycleChildScope =
-            CoroutineScope(owner.lifecycleScope.coroutineContext + lifecycleChildJob)
-    }
 
     fun clear() {
-        lifecycleChildJob.cancel()
```

Comment:
@jhavatar Is this cleanup now getting handled somewhere else? Does the CombustionService.onDestroy() handle cleaning up any coroutines potentially created in this module?